### PR TITLE
CLDR-15747 New API for getting alt attribute values

### PR DESCRIPTION
**Pull request by btangmu**
-New api/xpath/alt/hexId gets array of alt attribute values for specified xpath

CLDR-15747

- [ ] This PR completes the ticket.

### CODE REVIEW

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathAlt.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.unicode.cldr.icu.LDMLConstants;
+import org.unicode.cldr.util.*;
+import org.unicode.cldr.web.CookieSession;
+
+@Path("/alt")
```
Comment (srl295) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathAlt.java:19]:
```suggestion
@Path("/xpath/alt")
```

**Review by srl295 — CHANGES_REQUESTED** [review]
LGTM i wouldkeep the APIs together with others in xpath

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathAlt.java (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathAlt.java:19]: should that be true of /api/whatis as well? --> /api/xpath/whatis (then also need to revise in LookUp.vue)

Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathAlt.java]: for now I've just fixed it for alt

**Comment by btangmu** [comment]
I've made a bunch of further changes locally and it will be better to make a new PR from scratch.

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathAlt.java (hunk 1 above)
Comment (srl295) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathAlt.java]: probably good to move whatis also.

**Pull request closed without merging**